public/socialShare.js

```diff
@@ -0,0 +1,118 @@
+unlayer.registerTool({
+  name: 'social_share_tool',
+  label: 'Social Share',
+  icon: 'fa-share-alt',
+  category: 'contents',
+  type: 'whatever',
+  values: {},
+  options: {
+    default: {
+      title: null,
+    },
+    social_share_props: {
+      title: 'Propiedades',
+      position: 1,
+      options: {
+        social_share_size: {
+          label: 'Tamaño',
+          defaultValue: 'normal',
+          widget: 'dropdown',
+        },
+      },
+    },
+  },
+  renderer: {
+    Viewer: unlayer.createViewer({
+      render(values) {
+        var social_share_size = '90';
+
+        if (values.social_share_size === 'normal') {
+          social_share_size = '90';
+        } else if (values.social_share_size === 'small') {
+          social_share_size = '70';
+        } else if (values.social_share_size === 'big') {
+          social_share_size = '120';
+        }
```

Comment:
What do you think about trying to always use const to avoid mutations and simplify code? In this case, the ternary operator would help:


```suggestion
        const social_share_size = values.social_share_size === 'normal' ? '90'
        : values.social_share_size === 'small' ? social_share_size = '70'
        : values.social_share_size === 'big' ? social_share_size = '120'
        : throw new Error("unexpected value size");
```

Or if you want to make it safe and make honor to your previous default value:


```suggestion
        const values.social_share_size === 'small' ? social_share_size = '70'
        : values.social_share_size === 'big' ? social_share_size = '120'
        : '90';
```

Reply:
Great suggestion! Thx